.github/workflows/image.yml

```diff
@@ -0,0 +1,54 @@
+name: image
+on:
+  pull_request:
+  push:
+    branches: [master]
+jobs:
+  build-image:
+    strategy:
+      matrix:
+        arch: [amd64]
+    runs-on: ubuntu-latest
+    env:
+      # IMG_CACHE: ghcr.io/getsentry/symbolicator:${{ matrix.arch }}-latest
+      # IMG_VERSIONED: ghcr.io/getsentry/symbolicator:${{ matrix.arch }}-${{ github.sha }}
+      BUILDER_IMG_CACHE: ghcr.io/getsentry/symbolicator:${{ matrix.arch }}-builder
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - run: docker run --rm --privileged tonistiigi/binfmt --install arm64
+        if: matrix.arch == 'arm64'
+      - name: build
+        run: |
+          set -euxo pipefail
+          args=()
+          if docker pull -q "$BUILDER_IMG_CACHE"; then
+            args+=(--cache-from "$BUILDER_IMG_CACHE")
+          fi
+          docker buildx build \
+              "${args[@]}" \
+              --build-arg BUILDKIT_INLINE_CACHE=1 \
+              --platform linux/${{ matrix.arch }} \
+              --tag "$BUILDER_IMG_CACHE" \
+              --target symbolicator-build \
+              .
+      - name: push
+        run: |
+          set -euxo pipefail
+          docker login --username '${{ github.actor }}' --password '${{ secrets.GITHUB_TOKEN }}' ghcr.io
+          docker push "$BUILDER_IMG_CACHE"
+        if: github.event_name != 'pull_request'
+
+  multiarch:
+    if: github.event_name != 'pull_request'
+    needs: [build-image]
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          set -euxo pipefail
+          docker login --username '${{ github.actor }}' --password '${{ secrets.GITHUB_TOKEN }}' ghcr.io
+          docker manifest create ghcr.io/getsentry/symbolicator:latest \
+            ghcr.io/getsentry/symbolicator:arm64-latest \
+            ghcr.io/getsentry/symbolicator:amd64-latest
+          docker manifest push ghcr.io/getsentry/symbolicator:latest
```

Comment:
this will fail with only one image -- and we can't use manifest-based images if we only have 1 architecture or it will break M1